js-exercises/abbreviate-string/abbreviateString.js

```diff
@@ -0,0 +1,14 @@
+// import { isString } from "util";
+
+function abbreviateString(str) {
+    if (typeof str !== 'string'){
+        throw 'Invalid argument! it must be a string';
```

Comment:
Check out different types of errors you can throw. In this case, `TypeError` seems appropriate to me. Not 100% sure though. 

Something like:
```js
throw new TypeError('Input must be a string');
```

Reply:
Yes, the type error looks more relevant, I did not new about that thanks to pointing it out I'll use it now :)